docs/reference/adobe/experience/customerJourneyManagement/messageexecution.schema.json

```diff
@@ -28,6 +28,11 @@
                     "type": "string",
                     "description": "The parent Message ID of a MessageExecution originating this message."
                 },
+                "https://ns.adobe.com/experience/customerJourneyManagement/messageExecution/messagePublicationId": {
```

Comment:
messagePublicationId -> messagePublicationID (ID to keep it consistent with other ids)